docs/CHANGELOG.asciidoc

```diff
@@ -402,11 +434,11 @@ Visualization::
 == {kib} 5.6.1
 
 Important Note::
-* The 5.6.1 release of the Elastic Stack has an important fix for an issue 
-affecting users upgrading with indexes created in Elasticsearch 2.x and earlier 
-where Kibana becomes unavailable. We recommend that you upgrade to 5.6.1 to 
-avoid the issue. You can read the details of the problem and about mitigation 
-if you are already experiencing the issue in this https://discuss.elastic.co/t/upgrade-issue-with-elastic-stack-5-6-0-workaround-option-until-fix-is-available/100595[forum post]. If you are an Elastic Cloud user then Elastic Cloud support is working 
+* The 5.6.1 release of the Elastic Stack has an important fix for an issue
```

Comment:
It'd be good if we could get all the docs folks using the same editor rules, like removing whitespace at the end of lines, otherwise we're going to see an increasing number of these sorts of whitespace changes in every doc change. We had this problem a long time ago and we addressed it, but it seems to be creeping back.